server/lib/shared.py

```diff
@@ -17,6 +17,17 @@
 
 import server.lib.fetch as fetch
 
+# Maps enclosed place type -> places with too many of the enclosed type
+# Determines when to make batched API calls to avoid server errors.
+NEEDS_SPECIAL_HANDLING = {
```

Comment:
nit: BATCHED_CALL_PLACES or something that describes this special group of places. NEEDS_SPECIAL_HANDLING is very vague

also, curious why putting this constant in the shared file when it's specific to observation?

Reply:
leaving it in shared file was a holdover from when `point.py` also needed to use it. Now that it's specific to series, moved this to `series.py` and renamed to `BATCHED_CALL_PLACES`